impl/src/main/java/org/jboss/cdi/tck/tests/event/observer/method/StockWatcher.java

```diff
@@ -16,10 +16,14 @@
  */
 package org.jboss.cdi.tck.tests.event.observer.method;
 
+import jakarta.annotation.Priority;
 import jakarta.enterprise.context.Dependent;
 import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.inject.Alternative;
 
 @Dependent
+@Alternative
+@Priority(1)

```

Comment:
Are these needed? It was a selected bean before, no?

Reply:
Ah, it's just to be able to assert more facts about the bean, right?

Reply:
Exactly, I added that solely to be able to assert a few things about the bean :-)